x-pack/plugins/security_solution/server/endpoint/routes/trusted_apps/handlers.test.ts

```diff
@@ -224,6 +288,24 @@ describe('handlers', () => {
         )
       ).rejects.toThrowError(error);
     });
+
+    it("should return error when policy doesn't exists", async () => {
+      const mockResponse = httpServerMock.createResponseFactory();
+      packagePolicyClient.getByIDs.mockReset();
+      packagePolicyClient.getByIDs.mockResolvedValueOnce(PACKAGE_POLICIES_RESPONSE);
+
+      await createTrustedAppHandler(
+        createHandlerContextMock(),
+        httpServerMock.createKibanaRequest({ body: TRUSTED_APP_BY_POLICY }),
+        mockResponse
+      );
+
+      expect(appContextMock.logFactory.get('trusted_apps').error).toHaveBeenCalledWith(
```

Comment:
Suggestion: also add a check to ensure the correct http code was generated for this condition. SOmething like this (I think) should do it:


```typescript
expect(mockResponse.badRequest.toHaveBeencalledWith({ body: errorInstanceHere}
```

Reply:
done: https://github.com/elastic/kibana/pull/106110/commits/2f5dc946c1b4b0def4ce2659e074f961035425f5